Almost final version ;-) (#5)
inbo/n2khab-droogtemeetnet: 02_Methodiek.Rmd

> #Gebruikte methodieken

Reviewer: ```suggestion
# Methode
```

## Ontwerp van het meetnet

### Ruimtelijke selectie van rastercellen {#ruimte-sel-raster}

Idealiter liggen de meetlocaties ruimtelijk gebalanceerd, omdat dit de beste garantie biedt dat de gebeurtenissen die zich op deze locaties voordoen, representatief zijn voor het hele gebied.   
Men kan een ruimtelijk gebalanceerde set opbouwen door de verspreiding van de GT-groepen over een fijn GRTS-raster[^1] te leggen. 

> [^1]: GRTS = Generalized Random Tessellation Stratified

Reviewer: ```suggestion
[^1]: GRTS-design = Generalized Random Tessellation Stratified design.
Een GRTS-raster is hiervoor de basis.
```

1. De kans dat op een dergelijk punt zich een bestaand grondwatermeetpunt bevindt, is namelijk weinig realistisch. 
Het karakteriseren van een nieuwe locatie, zodat het kan ingezet worden in het meetnet, vergt verschillende jaren grondwatermetingen.
1. We hebben ook geen controle over de verdeling van de punten over de verschillende grondwatertypegroepen, die verschillend op droogte kunnen reageren.   

We kunnen om deze redenen geen zuiver ruimtelijk gebalanceerde set opbouwen. 

De drie hoofdfactoren waar we bij het ontwerp mee rekening houden, zijn:

- We willen zoveel mogelijk een ruimtelijk gebalanceerde keuze. 
De keuze van de grootte van een rastercel en bijgevolg van het aantal cellen, is hierbij sturend. 
Deze wordt bepaald door de ingeschatte grootte van het meetnet en door de trefkans op een (liefst actueel) meetpunt van grondwaterpeilen. 
Hoe kleiner de celgrootte, hoe beter de ruimtelijke balans kan zijn, omdat men minder vrij is om een locatie te kiezen, maar ook hoe kleiner de kans is om daar een geschikt meetpunt aan te treffen. 
Hier is het dus wat zoeken naar een goede maat. 
Concreet:  

>     - We maken gebruik van een set gekoppelde GRTS-rasters op een verschillende resolutie ([GRTSmh_diffres](https://doi.org/10.5281/zenodo.3354405)) die zijn afgeleid van het basisraster 'GRTSmaster_habitats' ([link](https://doi.org/10.5281/zenodo.2682323)).

Reviewer: Heb deze opgenomen in de literatuurlijst:

```suggestion
    - We maken gebruik van een set gekoppelde GRTS-rasters op een verschillende resolutie [@vanderhaeghe_grtsmh_diffres_2019] die zijn afgeleid van het basisraster 'GRTSmaster_habitats' [@onkelinx_grts_2019].
```

Er zou dan aan geen enkele cel een meetpunt kunnen toegewezen worden.

Om dit euvel te ondervangen, werd een iteratie uitgevoerd. 
Na de eerste ronde worden per GT-groep al de bekomen aantallen opgeteld en vergeleken met het gewenste aantal. 
Is dat aantal te klein, dan wordt de benodigde oppervlakte een klein beetje verkleind (bijv. 9.9 ha per meetpunt) en wordt de berekening opnieuw uitgevoerd en het totaal aantal gecheckt.
Dit gaat door tot wanneer het gewenst aantal bereikt wordt.
De variatie in oppervlakten bepaalt de grootte van de verkleining. 
Is de sprong voldoende klein, zal het bekomen aantal hooguit lichtjes hoger dan het gewenste aantal uitvallen.

Figuur \@ref(fig:workflow-rastercellen) geeft een schematisch overzicht van de gevolgde werkgang.

(ref:workflow-rastercellen) werkgang selectie rastercellen. Blauw gekleurde cirkels zijn brongegevens. De rode gekleurde zijn genereerde data. Het groen gekleurde is een (tussentijds) output-resultaat. De gele vakken stellen handelingen/bewerkingen voor.

```{r workflow-rastercellen, fig.cap='(ref:workflow-rastercellen)'}
include_graphics(path = file.path("figures", "workflow_rasterselectie.png"))

> ```

Reviewer: Lijkt me OK, jij zult dit het beste kunnen beoordelen :wink:
Daarom enkel vormelijke opmerkingen:

- 'ga' is wat onduidelijk, bv. in 'ga in GRTS-rasters' lijkt het een werkwoord. Misschien tussen ' ' zetten, ...
- 'GIS' wordt het vaakst gebruikt voor dedicated software pakketten, maar sensu lato is de term zeker juist. Omdat het door die verwarring kan overkomen als 'ik heb hocus-pocus gedaan in arcgis' lijkt het me hier duidelijker om 'GIS:' te vervangen door 'geografische' of 'geospatiale'.

Reviewer: Zeker ergens in de methodiek opnemen: _de verwerking is volledig gebeurd met R versie 3.6.1_ , met daarbij een literatuurverwijzing naar R. Voor dat laatste, gebruik de output van `citation()`.

Voorts kun je best iets toevoegen als: _de code voor analyse en dit rapport is te vinden op de GitHub-repository [n2khab-droogtemeetnet](https://github.com/inbo/n2khab-droogtemeetnet)_.

Beide zaken kun je wellicht vooraan in methoden vermelden, bv. apart en direct onder de titel van hfst. 2 'gebruikte methodieken'.

+ je kunt daar verder verwijzen naar bijlage `\@ref(gebruikte-werkomgeving)`.

Author: OK in die zin je goede suggesties verwerkt

Dit kunnen locaties zijn die op grens liggen van habitatvlekken die tot een verschillende GT-groep behoren of het zijn locaties die in een habitatvlek gelegen zijn dat gekarteerd is als een complex van vegetaties die tot verschillende GT-groepen behoren.
In de praktijk wordt deze voorwaarde toegepast door enkel die habitatvlekken en/of peilbuizen te selecteren die voor 50% of meer toegewezen kunnen worden aan eenzelfde GT-groep en die op minstens `r bufferpb` m afstand liggen van een andere GT-groep.

[^2]: Normaal gesproken duidt in dit rapport de term meetpunt op de locatie eerder dan op het meetinstrument (peilbuis of piëzometer), maar soms slaat het op beide. Met peilbuis wordt in dit document ook een piëzometer bedoeld. 

Figuur \@ref(fig:workflow-selectiepb-part1) geeft een schematisch overzicht van de werkgang die gevolgd werd om na te gaan welke peilbuizen beschikbaar zijn.

(ref:workflow-selectiepb1) werkgang selectie rastercellen. Voor de betekenis van de symbolen zie figuur \@ref(fig:workflow-rastercellen).

```{r workflow-selectiepb-part1, fig.cap='(ref:workflow-selectiepb1)'}
include_graphics(path = file.path("figures", "workflow_pbselectie_part1.png"))

> ```

Reviewer: analoge opm als in vorige fig

Reviewer: 'één GT in een straal van 3 m': wellicht duidelijker indien je vooraan 'voorwaarde: slechts' toevoegt

Author: ok, figuur werd in die zin aangepast

**Categorie 2**

De rastercellen van deze categorie zijn voor het meetnetontwerp een zegen.
Het aantal locaties met peilbuizen volstaat juist om te voldoen aan de vraag.
Alle locaties kunnen zonder verdere analyse in het meetnet worden opgenomen.

**Categorie 3**

Voor een rastercel van categorie 3 stelt er zich een 'luxe-probleem':
er is namelijk een overschot aan meetpunten met een peilbuis.
Uit dit aanbod moet nog het gewenste aantal meetpunten gekozen worden. 
Hiervoor verwijzen we naar paragraaf \@ref(sel-habvlek).

_B. De meetpunten met een peilbuis zijn kwalitatief niet gelijkwaardig_

In werkelijkheid zijn niet alle meetpunten even geschikt. 

We noemen een meetpunt geschikt, indien er hiervoor een droogte-indicator kan berekend worden.
Het berekenen van deze indicator op een meetpunt vergt een ononderbroken meetreeks van _dagelijkse_ peilen en dit over een stabiele, langdurige periode (een paar tiental jaren) . 
De beschikbare terreinwaarnemingen in Watina zijn meestal ontoereikend. 
Door middel van een tijdreeksanalyse (zie \@ref(tijdreeksanalyse)) kunnen de tijdreeksen met modelgegevens aangevuld worden, waardoor het wel mogelijk wordt om de indicator te berekenen.

> Het zal bijgevolg op basis van de tijdreeksanalyse zijn dat meetpunten als geschikt (er is tijdsmodel mogelijk) of niet geschikt (een model is niet mogelijk) zullen bevonden worden.

Reviewer: ```suggestion
Het zal bijgevolg op basis van de tijdreeksanalyse zijn dat meetpunten als geschikt (een tijdsmodel is mogelijk) of niet geschikt (een tijdsmodel is niet mogelijk) zullen bevonden worden.
```

Het vergt veel tijd om op alle beschikbare meetpunten een tijdreeksanalyse uit te voeren.

> We willen dit beperken tot de *relatief (beschouwd op het niveau van combinatie rastercel en GT-groep)* kwalitatief beste meetreeksen. 

Reviewer: ```suggestion
We beperken dit daarom tot de *relatief (beschouwd op het niveau van combinatie rastercel en GT-groep)* kwalitatief beste meetreeksen. 
```

We leggen geen absolute minimum-criteria op. Indien in een rastercel voor een bepaalde GT-groep de langste tijdreeks één jaar is, dan zal deze peilbuis toch voor een tijdreeksanalyse geselecteerd kunnen worden. De kans wordt uiteraard kleiner dat met deze beperkte data een betrouwbaar model kan gebouwd worden.

> We delen de meetpunten op in een aantal kwaliteitsklassen op basis van volgende **[basiskwaliteitscriteria]{#basiskwaliteitscriteria}** :

Reviewer: ```suggestion
We delen de meetpunten daarom op in een aantal kwaliteitsklassen op basis van volgende **[basiskwaliteitscriteria]{#basiskwaliteitscriteria}** :
```

> (@) De meetpunten hebben vanaf 2000 voor minstens één jaar een goede tijdreeks[^3], die een toelaat een lg3 te berekenen.

Reviewer: ```suggestion
(@) De meetpunten hebben vanaf 2000 voor minstens één jaar een goede tijdreeks[^3], die een toelaat een zogenaamde lg3-waarde[^3] te berekenen.
```

(@) Meetpunten met een _recente_ kwalitatief goede tijdreeks primeren boven meetpunten met een oudere tijdreeks. 

> De meetpunten worden hierbij gerangschikt volgens aflopend jaartal (eindjaar) van de laatst beschikbare kwalitatief goede tijdreeks. 

Reviewer: inspringen:
```suggestion
  De meetpunten worden hierbij gerangschikt volgens aflopend jaartal (eindjaar) van de laatst beschikbare kwalitatief goede tijdreeks. 
```

> Meetpunten worden als evenwaardig beschouwd als het verschil niet groter dan **`r toelaatbare_spreiding_jaren`** jaar is.

Reviewer: inspringen:
```suggestion
  Meetpunten worden als evenwaardig beschouwd als het verschil niet groter dan **`r toelaatbare_spreiding_jaren`** jaar is.
```

(@) Meetpunten met een _langere_ tijdreeks, t.t.z. een groter aantal kwalitatief goede meetjaren, krijgen ook een hogere prioriteit.

> Meetpunten waarvan het aantal niet meer dan **`r toelaatbaar_verschil_lengte_tijdreeks`** jaar van elkaar verschilt, worden als evenwaardig beschouwd.

Reviewer: inspringen:
```suggestion
  Meetpunten waarvan het aantal niet meer dan **`r toelaatbaar_verschil_lengte_tijdreeks`** jaar van elkaar verschilt, worden als evenwaardig beschouwd.
```

[^3]: Een kwalitatief goed meetjaar wordt hier gedefinieerd als een hydrologisch jaar waarvoor een lg3 kan berekend worden. Dit houdt in dat er minstens 20 goed gespreide metingen per jaar zijn. Het verschil tussen twee metingen mag niet groter zijn dan 30 dagen en de eerste en laatste meting moet resp. in april en maart (van het daaropvolgende kalenderjaar) vallen. 
lg3 =  het gemiddelde van de drie laagste waterpeilen binnen een hydrologisch jaar (1 april - 31 maart), waarbij er tussen deze drie metingen minstens 14 dagen verschil ligt. -->

<!-- De extreem droge zomers van 2017 en 2018 worden niet meegenomen in de berekening. Het is namelijk nu nog onduidelijk (door mogelijke naijling) of de actuele vegetatie geen nadelige effecten van deze twee zomers heeft ondervonden. -->

<!-- Nemen we ook het kwaliteitsaspect mee in de beoordeling, dan wordt de categorie-indeling van rastercellen complexer.  -->
<!-- Op basis van de kwaliteit van de tijdreeksen kunnen dan per GT-groep de rastercellen in één of meer van de volgende categorieën opgedeeld worden. Deze categorieën sluiten elkaar niet mutueel uit (zie onder). -->

> **_vraag: wat te doen met bestaande meetlocaties die actueel niet geschikt zijn? Worden deze weer geactiveerd (gevolgd in deze versie) of geven we voorrang aan de ruimtelijke balans ?_**

Reviewer: ```suggestion
**_vraag: wat te doen met bestaande meetlocaties die actueel niet geschikt zijn? Worden deze weer geactiveerd (gevolgd in het huidige voorstel) of geven we voorrang aan de ruimtelijke balans ?_**
```

Alleen bij rastercellen van cat. 1 is er een nood aan extra meetpunten. Hoe potentieel geschikte habitatvlekken geselecteerd kunnen worden, wordt in \@ref(sel-pot-habvlek) nader toegelicht.
In categorie 1 wordt, ook als we rekening houden met het al dan niet geschikt zijn van een meetpunt, elk eventueel beschikbaar meetpunt geselecteerd.

Het onderscheiden van kwaliteitsverschillen heeft vooral een impact op de behandeling van de twee andere categorieën. 

> De grootte van deze clusters wordt dan vergeleken met het gewenste aantal gewenste meetlocaties voor een rastercel. 

Reviewer: ```suggestion
De grootte van deze clusters wordt dan vergeleken met het aantal gewenste meetlocaties voor een rastercel, per GT-groep. 
```

Kan het resterende aantal ingevuld worden met de som van het aantal peilbuizen in de x beste clusters, dan rekenen we die rastercel tot cat. 2. 
In het andere geval rekenen we ze tot cat. 3.
Voor deze laatste groep zal nog een selectie van peilbuizen dienen te gebeuren.

```{r voorbeeld-selectie-part1, fig.cap="Cijfervoorbeeld van de gevolgde workflow bij de selectie", , fig.width=8, fig.height=8, eval = FALSE}
  include_graphics(path = file.path(".","figures", 
                                    "voorbeeld_selectie_part1.png")) 

> ```

Reviewer: Witregels gebruikten om zaken te scheiden! Ik vermoed dat het ontbreken van een witregel onder r. 217 (scheiden van 2 chunks) ertoe leidde dat beide figuren niet getoond worden.

Het tweede doel is in feite een bijkomende selectiecriterium. Indien uit een tijdreeksanalyse blijkt dat de waterstanden van een meetpunt systematisch dalen of stijgen, leent dat meetpunt zich niet goed voor het berekenen van een droogte-indicator. 
Met een systematische daling of stijging bedoelen we een lineaire trend die niet door externe factoren (opstuwen of graven greppels, waterwinning) kan verklaard worden. 
Indien een tijdreeks een lineaire trend vertoont, wordt het meetpunt als momenteel ongeschikt beschouwd.

Voor de tijdreeksanalyse doen we beroep op het computerprogramma Menyanthes [@RN5899]. 
Voor het samenstellen van de verklarende reeksen (neerslag en evapotranspiratie) werd beroep gedaan op de data van de meetstations beheerd door KMI, VMM, HIC en KNMI. 
Voor de waterstandgegevens werd de Watina-databank geraadpleegd. 

Er werden geen kwaliteitseisen gesteld aan een meetreeks die deze a priori zou uitsluiten van een tijdreeksanalyse.
Het is, door het hoge aantal beschikbare meetpunten, echter niet doenbaar om een tijdreeksanalyse uit te voeren op alle meetpunten. 

> Wel werden alle meetreeksen van de meetpunten die behoren tot rastercellen van cat. 1B geanalyseerd. 

Reviewer: > cat. 1B

is niet langer gedefinieerd?

Voor de categorieën 2 en 3 bleef dat beperkt tot die meetpunten die behoren tot de hoogste kwaliteitsklassen die nodig zijn om tot het gewenst aantal meetpunten te komen.

De kwaliteit van een tijdreeks kan na een tijdreeksanalyse anders beoordeeld worden. 
Bijvoorbeeld een meetpunt met veel peilmetingen, die echter een duidelijke lineaire trend vertonen, is weinig geschikt om onmiddellijk in het meetnet te worden opgenomen.
Anderzijds komt een meetpunt met minder metingen, maar waarvoor een goed model kan gebouwd worden (zonder lineaire trend), wel in aanmerking om nu reeds in het meetnet te worden opgenomen.

Een meetreeks werd weerhouden als er een tijdsmodel kon voor samengesteld worden dat minstens 66% van de variatie in de waargenomen peilen (cfr. EVP, explained variance percentage, in de Menyanthes-software) kon verklaren en ook geen significante lineaire trend vertoont. 
Bij het tijdsmodel werden uitsluitend neerslag- en evapotranspiratiedata als verklarende reeksen gebruikt. 

Minstens twee derden van het peilverloop dient dus verklaard te kunnen worden door het weer.
Eén derde kan/mag door andere factoren beïnvloed worden (bijv. het naburige peil van een waterloop).

> Een [lineaire trend]{#lintrend} werd als significant beschouwd indien de stijging/daling per jaar (= `trend`), rekening houdend met de standaardafwijking hierop (= `se`), voldoet één van de volgende twee regels:

Reviewer: ```suggestion
Een [lineaire trend]{#lintrend} werd als significant beschouwd indien de stijging/daling per jaar (= `trend`) voldoet aan één van de volgende twee overschrijdingen in een éénzijdige statistische toets (met `se` de standaardfout van de trendschatting):
```

> $$trend - 1.96* se > 1(cm/jaar)$$ 

Reviewer: evenwaardig, maar nog vlotter te interpreteren:
```suggestion
$$trend > 1 (cm/jaar) + 1.96* se$$ 
```

of 

> $$trend + 1.96* se < -1(cm/jaar)$$

Reviewer: ```suggestion
$$trend < -1 (cm/jaar) - 1.96* se$$
```

> Er is dan 2.5% kans een trendwaarde die groter resp. kleiner is ten onrechte als een significante trend te bestempelen.

Reviewer: ```suggestion
Er is dan bij benadering, per statistische toets, 2.5% kans een trendwaarde die groter resp. kleiner is _ten onrechte_ als een significante trend te bestempelen (type I fout).
```

Bij de modelselectie kwam toch ook nog expertoordeel bij kijken. 
 
De gestelde kwaliteitseisen hebben alleen een weerslag op de categorie-indeling.
Ze leiden niet tot uitsluiting.

> In principe kan daarom elke peilbuis in categorie 1B terecht komen, zelfs is er maar één meting van bekend.

Reviewer: > categorie 1B

is niet langer gedefinieerd?

Anderzijds werd een model met een verklarende variantie beneden 66% soms nog weerhouden, indien bleek dat het model de lage grondwaterstanden goed kon modelleren.

Een lange tijdreeks met een trend < 1cm/jaar, maar waarbij de peilen maar weinig schommelen (kleine amplitude) kon toch owv deze trend afgekeurd worden (bijv. een tijdreeks waarbij over 15 jaar er een duidelijke stijging van 10 cm te zien is, terwijl de peilen slechts een amplitude van 40 cm hebben) en vice versa.

Het al dan niet weerhouden van een tijdreeks kan gevolgen hebben voor de categorie-indeling van een rastercel. 
Voor een rastercel kunnen immers na een tijdreeksanalyse meer of ook minder peilbuizen geschikt zijn. 
Daarom voeren we na de tijdreeksanalyse de categorisering nog een tweede keer (en mogelijk nog meerdere keren) uit, waarbij met de uitkomst van de tijdreeksanalyse rekening wordt gehouden. **_of verkiezen we de peilbuizen met de beste modellen ?_**

#### Bijkomende GRTS-analysen

Afhankelijk van de verhouding tussen het aantal gewenste en beschikbare peilbuizen, moeten er nog enkele bijkomende GRTS-analysen uitgevoerd worden.

> ##### Selecteren van potentiële geschikte habitatvlekken voor rastercellen waarvoor er te weinig peilbuizen bestaan (cat. 1) {#sel-pot-habvlek}

Reviewer: ```suggestion
##### Selecteren van potentieel geschikte habitatvlekken voor rastercellen waarvoor er te weinig peilbuizen bestaan (cat. 1) {#sel-pot-habvlek}
```

Blijkt bij inspectie op terrein dat de geselecteerde plaats toch niet aan de verwachtingen/criteria voldoet, dan wordt binnen dezelfde rastercel de rastercel die het eerst in rang volgt binnen een zoekstraal van 1 km bezocht.  

##### Selecteren van een meetpunt in een rastercel waarvoor meerdere kandidaten zijn (cat. 3) {#sel-habvlek}
Als het aantal beschikbare peilbuizen groter is dan het gewenste aantal en wanneer ook het aantal geschikte peilbuizen (cfr. tijdreeksanalyse) hiervan afwijkt, dient nog een bijkomende selectie van peilbuizen te gebeuren.
Indien het aantal geschikte peilbuizen dan groter is dan het gewenste aantal wordt enkel binnen deze groep verder gezocht.
In het andere geval omvat het alle peilbuizen.

Net zoals voor cat. 1 willen we de keuze zo aselect mogelijk doen. 
We doen hiervoor terug beroep op het fijnste GRTS-raster 'GRTSmaster_habitats' (celgrootte = 32 m).
Het/de meetpunt(en) gelegen in de deelrastercel(len) met het /de laagste rangnummer(s) zal/zullen dan geselecteerd worden.
Deze werkwijze laat ook gemakkelijk toe om reserve-punten te kiezen.
Het zijn de meetpunten die in rang volgen op de laagste rang van een gekozen punt.

> #### Samenvatting werkgang selectie peilbuizen

Reviewer: Ik raad aan om dit stuk te vervroegen, naar net vóór de titel 'Indeling van rastercellen in categorieën', en als volgt in te leiden.

```suggestion
#### Overzicht werkgang selectie peilbuizen

In de volgende paragrafen wordt in detail uiteengezet hoe we komen tot een selectie van peilbuizen per rastercel en GT-groep (en zonodig tot bijkomende nieuwe locaties), uitgaande van de waarden `B` en `G`.
Daarom schetsen we hier eerst een overzicht van deze workflow - zie ook figuur \@ref(fig:workflow-selectiepb-part2).

```

> Na de ruimtelijke selectie kennen we hoeveel meetpunten er per rastercel en per GT-groep gewenst zijn.

Reviewer: ```suggestion
Na de ruimtelijke selectie weten we hoeveel meetpunten er per rastercel en per GT-groep gewenst zijn.
```

We proberen die wensen zoveel mogelijk in te vullen met bestaande peilbuizen waarvan hun meetreeksen toelaten een droogte-indicator te berekenen.

> Hiertoe evalueren (zonder te selecteren) we de kwaliteit van de meetreeksen op basis van de mogelijkheid om voor minstens één jaar een lg3 te berekenen voor de metingen na 2000, van het aantal meetjaren en van de ouderdom van de meetreeks. 

Reviewer: ```suggestion
Hiertoe evalueren we (zonder te selecteren) de kwaliteit van de meetreeksen op basis van 1) de mogelijkheid om voor minstens één jaar een lg3^[3] te berekenen met de metingen na 2000, 2) het aantal meetjaren en 3) de ouderdom van de meetreeks. 
```

> De kwalitatief beste meetreeksen worden verder beoordeeld met een tijdreeksanalyse. Door het grote aantal peilbuizen was het niet mogelijk ze alle te analyseren. De voorkeur ging daarom uit naar de relatief beste reeksen. Een meetreeks wordt 'geschikt' bevonden als minstens 2/3 van de variatie door het weer kan verklaard worden en er in de reeks geen lineaire trend te zien is. Op basis van de tijdreeksanalyse worden de meetreeksen geherklasseerd in 'geschikt', 'niet 'geschikt' en 'onbekend' (indien niet behandeld). Er worden minstens zoveel peilbuizen geanalyseerd tot er evenveel of meer geschikte reeksen zijn dan het gewenste aantal, zoniet worden ze allemaal geanalyseerd.

Reviewer: (zinnen splitsen over regels)
```suggestion
De kwalitatief beste meetreeksen worden verder beoordeeld met een tijdreeksanalyse. Door het grote aantal peilbuizen was het niet mogelijk ze alle te analyseren. De tijdreeksanalyse gebeurt daarom eerst voor de beste meetreeksen. Een meetreeks wordt 'geschikt' bevonden als minstens 2/3 van de variatie door het weer kan verklaard worden en er in de reeks geen lineaire trend te zien is. Op basis van de tijdreeksanalyse worden de meetreeksen geherklasseerd in 'geschikt', 'niet geschikt' en 'onbekend' (indien niet behandeld). Er worden minstens zoveel peilbuizen geanalyseerd tot er evenveel of meer geschikte reeksen zijn dan het gewenste aantal, zoniet worden ze allemaal geanalyseerd.
```

Is een meetreeks 'geschikt', dan kan de peilbuis onmiddellijk in het meetnet opgenomen worden.
Is ze 'niet geschikt', dan kan dit pas na verloop van enkele jaren.

De rastercellen kunnen in drie categorieën (cat. 1- 3) verdeeld worden op basis van de verhouding tussen het aantal gewenste en beschikbare peilbuizen. 
Bij cat. 1 zijn er te weinig peilbuizen. De meetreeksen van alle peilbuizen werden geanalyseerd. Via een GRTS-procedure wordt het ontbrekend aantal aangevuld met habitatvlekken. 
In cat. 2 klopt het gewenste aantal met het beschikbaar aantal en hoeft er geen aansluitende analyse meer uitgevoerd te worden.
In cat. 3 zijn er meer peilbuizen beschikbaar. Afhankelijk van de verhouding tussen gewenste aantal en het aantal geschikte peilbuizen is er mogelijk een extra analyse nodig. 

> - Is het aantal geschikte peilbuizen groter dan wordt enkel binnen deze groep een GRTS-analyse uitgevoerd om het gewenste aantal te selecteren.

Reviewer: ```suggestion
- Is het aantal geschikte peilbuizen groter dan het gewenste aantal, dan wordt enkel binnen deze groep een GRTS-analyse uitgevoerd om het gewenste aantal te selecteren.
```

> - Is het aantal kleiner dan wordt er uit alle beschikbare peilbuizen via een GRTS-analyse het gewenste aantal geselecteerd.

Reviewer: ```suggestion
- Is het aantal kleiner dan het gewenste aantal, dan wordt er uit alle beschikbare peilbuizen via een GRTS-analyse het gewenste aantal geselecteerd.
```

> - Is het aantal gelijk dan hoeft er geen verdere analyse te gebeuren. Alle meetpunten kunnen in het meetnet worden opgenomen.

Reviewer: ```suggestion
- Is het aantal gelijk aan het gewenste aantal, dan hoeft er geen verdere analyse te gebeuren. Alle meetpunten kunnen in het meetnet worden opgenomen.
```

- Cat. 1: de vraag is groter dan het aanbod: 
er zijn geen meetpunten met een peilbuis of er is een gebrek aan
- Cat. 2: de vraag is gelijk aan het aanbod: 
elk meetpunt met een peilbuis wordt geselecteerd
- Cat. 3: de vraag is kleiner dan het aanbod: 
er is een overschot aan meetpunten met een peilbuis 
elk meetpunt wordt geselecteerd

**Categorie 1**
Vanuit het oogpunt van een meetnetontwerp kan een rastercel van categorie 1 over twee soorten meetpunten beschikken. 
Je hebt meetpunten zonder peilbuis en mogelijk ook meetpunten met een peilbuis. 
Alle meetpunten met een peilbuis worden zonder verder analyse geselecteerd.
Daarnaast zijn er (vele) kandidaat-meetpunten om te voorzien van een nieuwe peilbuis. 

_**Bovenstaande is een voorstel. Je zou ook kunnen beslissen dat enkel meetpunten met een peilbuis in aanmerking komen voor het meetnet.**_
_**Of het tegenovergestelde is ook een alternatief: dat voor de ruimtelijke balancering we geen voorkeur geven aan meetpunten met een peilbuis.**_
_**We vinden dat het huidige voorstel een goede balans vertoont tussen beide.**_

> Het gaat hier vooral over een tijdsinvestering: dergelijke meetpunten zullen pas na enkele jaren effectief meedraaien in het meetnet. 

Reviewer: deze zin slaat specifiek op de nieuw te installeren peilbuizen, maar die connectie is niet (meer) duidelijk nu

De eigenlijke kwaliteit van de tijdreeks wordt bepaald door een tijdreeksanalyse (zie \@ref(tijdreeksanalyse)). 

Een meetpunt kan op basis van de actueel beschikbare gegevens ongeschikt zijn, omdat de gegevens niet toelaten om deze indicator voldoende betrouwbaar te berekenen. 

Nemen we ook het kwaliteitsaspect mee in de beoordeling, dan wordt de categorie-indeling van rastercellen complexer. 

We kunnen niet alleen meerdere subcategorieën onderscheiden, de behandeling van de meetpunten in een rastercel wordt ook minder éénduidig.
In het geval dat alle peilbuislocaties gelijkwaardig waren, bevatten alleen rastercellen van categorie 1 twee soorten meetpunten.
Maken we een onderscheid in kwaliteit van de meetreeksen, dan zijn er meer toestanden mogelijk.
We lichten dit toe aan de hand van een tweetal voorbeelden.

Voorbeeld 1: in een rastercel zijn drie meetpunten gewenst. 
Er zijn momenteel maar twee peilbuizen beschikbaar, één met een goede tijdreeks en één met een slechte. 
De drie locaties vragen elk een andere benadering. 

> Voor het meetpunt zonder peilbuis, moet op zoek gegaan worden naar de meest presentatieve keuze. 

Reviewer: ```suggestion
Voor het meetpunt zonder peilbuis, moet op zoek gegaan worden naar de meest representatieve keuze. 
```

Reviewer: hier en ook elders: 'meetpunt zonder peilbuis' lijkt te suggereren dat je de locatie ervan al kent ('meetpunt'). Maar is vermoedelijk niet zo. Misschien duidelijker om te spreken van een 'nog ontbrekend meetpunt', ...?

De twee andere meetlocaties kunnen opgenomen worden, maar de ene locatie komt in de wachtrij te zitten, terwijl met de andere effectief al van start kan gegaan worden. 

> Om dit technisch te kunnen ondervangen, zal de rastercel tot drie categorieën gerekend worden (1A, 1Ba en 2).

Reviewer: Mooi verwoord :+1:

Voorbeeld 2: in een rastercel zijn drie meetpunten gewenst. 
Er zijn vijf peibuizen beschikbaar, twee met kwaliteitsvolle tijdreeksen, de drie andere met tijdreeksen van onvoldoende kwaliteit. 

> Deze rastercel zal behoren tot categorie 2 voor de twee peilbuizen van goede kwaliteit en tot categorie 1B: voor het derde meetpunt moet uit de drie peilbuizen met onvoldoende kwaliteit er één nog gekozen worden.

Reviewer: Misschien nog iets als '(zie verder)' toevoegen aangezien de cat-definitie nog komt.

> [^2]: lg3 =  het gemiddelde van de drie laagste waterpeilen binnen een hydrologisch jaar (1 april - 31 maart), waarbij er tussen deze drie metingen minstens 14 dagen verschil ligt.

Reviewer: > [^2]:

voor de zekerheid dat je het weet: het mogen ook labels zijn ipv nummers. Voetnootnummering gebeurt automatisch

Voor een meetpunt dat voldoet aan deze criteria kan een gemiddeld laagste grondwaterstand (GLG) berekend worden. 

Op basis van de kwaliteit van de tijdreeksen kunnen dan per GT-groep de rastercellen in (één of meer van de volgende) vier categorieën opgedeeld worden:

- **Cat. 1A**: de vraag is groter dan het aanbod, er is geen aanbod: aan de vraag kan enkel voldaan worden door nieuwe meetpunten te introduceren.
- **Cat. 1B**: de vraag is groter dan het aanbod van peilbuizen van goede kwaliteit: er is een gebrek aan geschikte meetpunten, de vraag kan wel ingevuld worden met actueel ongeschikte meetpunten.
Op basis van het aantal ongeschikte meetpunten, wordt nog een opdeling gemaakt:
- **Cat. 1Ba**: het aantal ongeschikte meetpunten is onvoldoende tot juist voldoende om alle gewenste locaties te kunnen invullen.
- **Cat. 1Bb**: het aantal ongeschikte meetpunten is groter dan het aantal gewenste locaties: er zal hiertussen nog een keuze moeten gemaakt worden.
- **Cat. 2**: de vraag is groter of gelijk aan het aanbod van peilbuizen van goede kwaliteit: er zijn geschikte meetpunten, juist voldoende of te weinig: elk meetpunt wordt geselecteerd en kan onmiddellijk ingezet worden.

> - **Cat. 3**: de vraag is kleiner dan het aanbod van peilbuizen van goede kwaliteit: er is een overschot aan geschikte meetpunten

Reviewer: op zich duidelijk verwoord :+1: 
Wel blijft het voorlopig nog een mysterie waarom een rastercel tot meerdere categorieën zou kunnen behoren

Reviewer: het begint me een beetje te dagen, na heen en weer kijken naar je twee voorbeelden. In feite ga je groepen van meetpunten binnen een rastercel onderscheiden, iets in de richting van:

- er zijn x goede
- er zijn y 'slechte'
- er zijn z - x goede tekort en z-x kan > of < zijn dan y, en z-x kan ook < 0

Reviewer: Zie nog [deze opmerking](https://github.com/inbo/n2khab-droogtemeetnet/pull/5#discussion_r344430340).

Reviewer: Op basis van latere lezing (zie latere review), voeg ik hier een tentatieve definitie in op niveau van de rastercel, met behulp van x, y, z (z = totaal aantal nodig), en geënt op je bovenstaande beschrijving

- 1A: x < z en y = 0
- 1B: x < z en y > 0
  - 1Ba: x + y <= z
  - 1Bb: x + y > z
- 2: x <= z en y >= 0. Waarom de overlap met de voorgaande?
  - deze definitie is in overeenstemming met (regel 792 van Rmd 03): "_Als de kwaliteit van de tijdreeks van een aantal peilbuizen voldoende is om onmiddellijk in een meetnet te kunnen worden opgenomen, wordt de cel ook tot categorie 2 gerekend._" Echter: waarom deze complexiteit van iets tot 2 categorieën rekenen? De term 'categorieën' impliceert m.i. reeds 'wederzijds uitsluitend', dus geen overlap.
  - anderzijds, onder de kop 'het opleggen van bijkomende kwaliteitscriteria' (Rmd 03) staat dan weer: "_Er kan dan een groep cellen onderscheiden worden waarbij het gewenste aantal meetpunten juist overeenstemt met het aantal beschikbare peilbuizen die gelijkwaardig van kwaliteit zijn. Deze rastercellen behoren tot categorie 2._" Dat wijst op een definitie die veel logischer past in het rijtje: **x = z**. Dus: consequent maken. Als je 't mij vraagt, deze categorie beperkten tot x = z; overlap lijkt me onnodig complex.
- 3: x > z

_Het lijkt me een goed idee om, louter voor de uitleg, de peilbuizen van een GTgroep binnen een cel in klassen in te delen (goede, slechte, evt. meer, alleen als dat nodig is). Vervolgens de celcategorieën definiëren op basis van de onderlinge relatie tussen de frequentie van die klassen (cf. x, y hierboven) en op basis van de relatie met het gewenste aantal meetpunten (z hierboven) eenvoudige formules als hierboven._ Tekstueel is het m.i. moeilijker te volgen dan met een formulevorm. Zit daar iets in?

Voorts, als volgens de definities x = z en x > z geen cellen tot cat 2 en 3 behoren, dan kan dit bij de uitwerking zo gesteld worden en hoef je alleen verder met 1A en 1B (wat je alleszins doet), zonder die laatste twee verder in relatie te brengen met 2 en 3. (In het andere geval: dan was me niet duidelijk dat dit zo is).

Zie ook een aantal opmerkingen bij de review in PR #3 .

en 
$$trend + 2* sd \geq -1(cm/jaar)$$

> Bij de modelselectie kwam toch ook nog expertoordeel bij kijken. 

Reviewer: ```suggestion
Bij de modelselectie kwam toch ook nog expertoordeel kijken. 
```

Er werden geen kwaliteitseisen gesteld aan een meetreeks die deze a priori zou uitsluiten van een tijdreeksanalyse. 
De gestelde eisen hebben alleen een weerslag op de categorie-indeling. 

> In principe kan elke peilbuis in categorie 1B terecht komen, zelfs is er maar één meting van bekend.

Reviewer: de categorieën lijken volgens deze zin meetpunt-gebonden (rekening houdend met andere meetpunten in rastercel), niet op niveau rastercel zoals eerder gedefinieerd. Dat is niet in overeenstemming met de definitie, noch hoe het uitgevoerd wordt. Dus consistent maken met elders.

En zo geschiedde ook.

> Deze meetreeksen werden manueel afgekeurd. Sommige van deze tijdreeksen hadden bijna een perfecte fit, maar het was toch onbetrouwbaar om hiermee waarden te simuleren over een relatief lange tijdsperiode. 

Reviewer: > Er werden geen kwaliteitseisen gesteld aan een meetreeks die

> werden manueel afgekeurd.

Op een of andere manier heeft de 'manuele afkeuring' een bepaalde regel gevolgd. Wellicht kan die regel code-matig geïmplementeerd worden zodat een pre- of postselectie voor/van tijdsreeksanalysen kan gebeuren?

Anderzijds werd een model met een verklarende variantie beneden 66% soms nog weerhouden, indien bleek dat het model de lage grondwaterstanden goed kon modelleren.

> Een tijdreeks met een trend < 1cm/jaar, maar waarbij de peilen maar weinig schommelen (kleine amplitude) kon toch afgekeurd worden en vice versa.

Reviewer: ```suggestion
Een tijdreeks met een (absolute) trend < 1cm/jaar, maar waarbij de peilen maar weinig schommelen (kleine amplitude) kon toch afgekeurd worden en vice versa.
```
